frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/simulated-region-overview-behavior-tab.component.ts

```diff
@@ -41,12 +44,28 @@ export class SimulatedRegionOverviewBehaviorTabComponent
         private readonly store: Store<AppState>
     ) {}
 
-    ngOnInit(): void {
+    async ngOnInit() {
         if (globalLastBehaviorType !== undefined) {
             this.selectedBehavior = this.simulatedRegion.behaviors.find(
                 (behavior) => behavior.type === globalLastBehaviorType
             );
         }
+        if (this.initialTransferOptions) {
+            this.selectedBehavior = this.simulatedRegion.behaviors.find(
+                (behavior) => behavior.type === 'transferBehavior'
+            );
+            if (!this.selectedBehavior) {
+                await this.exerciseService.proposeAction({
+                    type: '[SimulatedRegion] Add Behavior',
+                    simulatedRegionId: this.simulatedRegion.id,
+                    behaviorState:
+                        simulationBehaviorDictionary.transferBehavior.behaviorState.create(),
+                });
```

Comment:
You should add something like:
```suggestion
                const response = await this.exerciseService.proposeAction({
                    type: '[SimulatedRegion] Add Behavior',
                    simulatedRegionId: this.simulatedRegion.id,
                    behaviorState:
                        simulationBehaviorDictionary.transferBehavior.behaviorState.create(),
                });
                if (!response.success) return
```

Reply:
I do not see why I would return if I couldn't add the behavior... The current Code the switches to no behavior at all what is expected. A return would just crash the application in that case. One could then try to go to the globalLastBehavior . But I wouldn't suggest that since that would seem even more unintuitive than showing no behavior

Reply:
Ah, makes sense